components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -129,11 +129,13 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
                     <WorkspaceLogs classes="h-full w-full" logsEmitter={logsEmitter} errorMessage={error?.message} />
                 </Suspense>
             </div>
-            <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
-                {prebuild && <PrebuildStatus prebuild={prebuild} />}
-                <div className="flex-grow" />
-                {props.children}
-            </div>
+            {prebuild && (
+                <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2 mb-6">
```

Comment:
It seems the new `mb-6` now make the "Prebuild Details" view off again :cry: : 
![image](https://user-images.githubusercontent.com/32448529/177194941-ca4c8633-5457-4e95-8d36-6d2b572b6f9f.png)

Seems like we need to always test those three (1. PB details, 2. wait for PB, 3. manually trigger PB) in combination. :shield:

Reply:
Good catch 😭 Need to remind myself to test all three views! Will fix.

Reply:
Works now! :+1: